src/components/LeftDrawerEvent/LeftDrawerEvent.tsx

```diff
@@ -160,14 +159,10 @@ const leftDrawerEvent = ({
             }}
           >
             <div className={styles.imageContainer}>
-              {userImage && userImage ? (
-                <img src={userImage} alt={`Profile Picture`} />
-              ) : (
-                <img
-                  src={`https://api.dicebear.com/5.x/initials/svg?seed=${firstName}%20${lastName}`}
-                  alt={`Dummy User Picture`}
-                />
-              )}
+              <img
+                src={`https://api.dicebear.com/5.x/initials/svg?seed=${firstName}%20${lastName}`}
```

Comment:
This does not solves the issue, what will happen when a user will have a profile picture uploaded and he gets to see this initial image from Dicebear ?

Reply:
@rishav-jha-mech I have updated the code.